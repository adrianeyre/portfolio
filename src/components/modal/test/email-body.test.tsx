/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';

import IEmailBodyProps from '../interface/email-body-props';
import EmailBody from '../email-body';

describe('EmailBody', () => {
	const mockRouteProp: any = jest.fn();

	const defaultProps: IEmailBodyProps = {
		item: undefined,
		closeModal: mockRouteProp,
	};

	it('Should render correctly', () => {
		const emailBodyComponent = render(<EmailBody { ...defaultProps }/>);

		expect(emailBodyComponent).toMatchSnapshot();
	});
});
