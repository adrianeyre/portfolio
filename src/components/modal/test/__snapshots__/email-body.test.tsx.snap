// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailBody Should render correctly 1`] = `ShallowWrapper {}`;
